### PR TITLE
add rai-core-flask

### PR DESCRIPTION
**Pull request by bgalvao**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bgalvao** [comment]
@janjagusch linter is happy and all tests passed

**Review by janjagusch — APPROVED** [review]
Looks good. @xhochy, another one for you to take a final look at when you have the time. 🙂 

Generally, the design of this build is motivated by the same constraints as described [here](https://github.com/conda-forge/staged-recipes/pull/19439#pullrequestreview-1020171186).

**Comment by bgalvao** [comment]
@janjagusch this package is not depending on any other PR too

File: recipes/rai-core-flask/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - flask
+    - flask-cors
+    - gevent ==21.12.0
+    - greenlet ==1.1.2
+    - ipython ==7.31.1  # [py>36]
+    - itsdangerous ==2.0.1
```
[the remark this answers is not in the document]
Comment (bgalvao) [reply recipes/rai-core-flask/meta.yaml:26]: good catch!

[the remark this answers is not in the document]
Comment (bgalvao) [reply recipes/rai-core-flask/meta.yaml]: after [0209f9b](https://github.com/conda-forge/staged-recipes/pull/19445/commits/0209f9b71771abf1afabac389089d02131f57f61):

```
rai-core-flask 0.4.0 has requirement ipython==7.31.1; python_version > "3.6", but you have ipython 7.33.0.
rai-core-flask 0.4.0 has requirement itsdangerous==2.0.1, but you have itsdangerous 2.1.2.
Tests failed for rai-core-flask-0.4.0-py37h89c1867_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for rai-core-flask-0.4.0-py37h89c1867_0.tar.bz2 - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: rai-core-flask-0.4.0-py37h89c1867_0.tar.bz2
```

[the remark this answers is not in the document]
Comment (janjagusch) [reply recipes/rai-core-flask/meta.yaml]: hmm, you're right. we should probably leave revert https://github.com/conda-forge/staged-recipes/pull/19445/commits/0209f9b71771abf1afabac389089d02131f57f61 and leave everything as close as possible to the upstream version.

but maybe we could open a PR in `rai-core-flask` to adjust their `requirements.txt` (or not use their `requirements.txt` as their `install_requires`). that shouldn't be critical to get this PR merged, though.

[the remark this answers is not in the document]
Comment (janjagusch) [reply recipes/rai-core-flask/meta.yaml]: https://github.com/microsoft/responsible-ai-toolbox/pull/1578/files

**Pull request closed without merging**